src/edu/csus/ecs/pc2/ui/MultipleDataSetPane.java

```diff
@@ -751,4 +789,26 @@ public void actionPerformed(ActionEvent e) {
         }
         return chckbxStopOnFirstFailedTestCase;
     }
+
+    private JCheckBox getLoadSamplesFirstCheckbox() {
+        if (loadSamplesFirstCheckbox == null) {
+            loadSamplesFirstCheckbox = new JCheckBox("Load Sample Data");
```

Comment:
The above constructor (`JCheckBox(String)`) creates a checkbox which is initially _unselected_.  Since we agreed to change the default YAML Load condition to `load-sample-judges-data: true`,  this checkbox should be constructed to be automatically _selected_; that is, use the constructor `JCheckBox("...", true)`.  I consider this a SHOW STOPPER; we should not merge GUI code which is inconsistent with what's already merged to Develop.  (Yes, I'm aware that the requirement has changed since this code here was written; that's why I'm pointing out that it should be updated).

Also:  I think the checkbox label "Load Sample Data" is confusing.  When I see that as a user, I'm likely to think (indeed, what I _did_ think the first time I saw it) is that checking the box indicates that I want to load the _sample_ data _instead of the secret data_.  I suggest changing the label on the checkbox (i.e., in the above constructor) to something like `Include sample data` or "Also load sample data", so that it's clear that the checkbox means that the sample data is _also_ to be included.

Reply:
pushed fix.